CONTRIBUTING.md

```diff
@@ -12,3 +20,10 @@ for general information about the project.
 
 After making any changes to .proto files make sure to generate all
 implementation by running `make gen-all`.
+
+## Further Help
+
+If you have any questions or need assistance while contributing,
+feel free to reach out to the [`otel-specification`](https://cloud-native.slack.com/archives/C01N7PP1THC) Slack channel. 
+
+View meeting notes of previous SIG calls too [here](https://github.com/open-telemetry/opentelemetry-specification) to stay up to date.
```

Comment:
This link doesn't have meeting notes. Meeting notes are in this [google doc](https://docs.google.com/spreadsheets/d/1SYKfjYhZdm2Wh2Cl6KVQalKg_m4NhTPZqq-8SzEVO6s/edit?gid=0#gid=0) as [documented here](https://github.com/open-telemetry/community/?tab=readme-ov-file#governing-bodies).